O365DataRetriever.ps1

```diff
@@ -942,59 +958,66 @@ $ConnectButton.Add_Click( {
 
                 # Save the file...
                 if ($SaveGAFileDialog.ShowDialog() -eq 'OK') {
-                    (Get-MsolRoleMember -RoleObjectId $GARoleObjectId) | Select-Object DisplayName, EmailAddress, IsLicensed | Export-Csv $($SaveGAFileDialog.filename) -NoTypeInformation -Encoding UTF8
+                    $script:GlobalAdmins | Export-Csv $($SaveGAFileDialog.filename) -NoTypeInformation -Encoding UTF8
```

Comment:
Is this working? Have you tested it?

Reply:
Looks good for me.
![image](https://user-images.githubusercontent.com/16267927/44922828-c0dfab80-ad46-11e8-9338-58a82c4b2ee2.png)